src/commands/dev/edge/setup.rs

```diff
@@ -38,10 +39,20 @@ pub(super) fn upload(
         (Vec::new(), None, None)
     };
 
-    let session_config = get_session_config(deploy_config);
+    let session_config = get_session_config(&deploy_config.http_routes);
     let address = get_upload_address(target);
 
-    let script_upload_form = upload::form::build(target, asset_manifest, Some(session_config))?;
+    let actor_namespaces = actors::convert_namespace_names_to_ids(
```

Comment:
This is wrong in retrospect, as it tries to upload every implemented actor namespace when uploading to wrangler dev.

The edge-preview api does not consider them either, so we should probably remove this and pass an empty vec.